hello-world-notification-microservice/README.md

```diff
@@ -72,4 +72,43 @@ we will receive the following message as we have specified to transform the filt
   "source": {"id":"<DEVICE ID>"},
   "type": "c8y_Speed"
 }
-```
\ No newline at end of file
+```
+
+## Device creation and measurement sending
+
+The included [Python script](demo/measurements.py) has everything to get you started with the microservice and has example code for device creation, deletion and sending out measurements.
+To use the script you will need to provide authentication details, edit the script and provide values to the following fields:
+
+```python
+_tenant_id = ''
+_username = 'admin'
+_password = ''
+_platform_url = ''
+```
+
+To use, run the script with one of the options:
+
+### To create a device
```

Comment:
Can/should we make it clearer that the second line is output?

Reply:
That maybe goes for all 3. If other topics in the docs take this approach, then leave as is.

 I would guess (from ops-docs) it might be best to show the command specification with placeholders in bold (or similar) text first and then show the console as you have as an example (with actual values rather than placeholders).
 
 Happy to be argued with